README.md

```diff
@@ -2,47 +2,60 @@
 
-> 🤖 Automated multi-language coverage reporter for [Codacy](https://www.codacy.com/).  
-> For more information about Codacy see our [5 minutes quickstart](https://docs.codacy.com/getting-started/codacy-quickstart/).
+GitHub Action for [uploading coverage reports to Codacy](https://docs.codacy.com/coverage-reporter/#uploading-coverage) on all commits and pull requests, allowing you to track code coverage on your repository and to flag pull requests that cause coverage to drop or add code that isn't tested.
 
-This action runs the [codacy-coverage-reporter](https://github.com/codacy/codacy-coverage-reporter) for all commits and pull requests with your Codacy configuration.
+[Codacy](https://www.codacy.com/) is an automated code review tool that makes it easy to ensure your team is writing high-quality code by analyzing more than 40 programming languages such as PHP, JavaScript, Python, Java, and Ruby. Codacy allows you to define your own quality rules, code patterns and quality settings you'd like to enforce to prevent issues on your codebase.
 
-## Usage
+<br/>
 
-Create a new workflow `.yml` file in the `.github/workflows/` directory.
+<a href="https://www.codacy.com" target="_blank"><img src="images/codacy-logo.svg" alt="Codacy" width="400"/></a>
 
-### .github/workflows/codacy-coverage-reporter.yml
+<br/>
 
-```yml
-name: codacy-coverage-reporter
+## Uploading coverage to Codacy
 
-on: ["push"]
+Before setting up Codacy Coverage Reporter GitHub Action you must have tests and use tools to [generate coverage reports](https://docs.codacy.com/coverage-reporter/#generating-coverage) for the source code files in your repository.
 
-jobs:
-  codacy-coverage-reporter:
-    runs-on: ubuntu-latest
-    name: codacy-coverage-reporter
-    steps:
-      - uses: actions/checkout@v2
-      - name: Run codacy-coverage-reporter
-        uses: codacy/codacy-coverage-reporter-action@v1
-        with:
-          project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
-          # or
-          # api-token: ${{ secrets.CODACY_API_TOKEN }}
-          coverage-reports: cobertura.xml
-```
+The Codacy Coverage Reporter GitHub Action:
 
-> Get the latest version from the [Releases page](https://github.com/codacy/codacy-coverage-reporter-action/releases).
+-   Uploads coverage report data to Codacy.
+-   Prints the coverage report upload status on the console, which is visible on the workflow panel of the GitHub Action.
+-   Fails the workflow if code coverage falls below your project’s [quality settings](https://docs.codacy.com/repositories-configure/adjusting-quality-settings/#gates).
 
-## Workflow options
+To upload coverage to Codacy using the GitHub Action with default settings:
```

Comment:
Which are the "default settings"? Is there an option to use other settings?

Reply:
There are some extra configurations, but I have to admit that the "with default settings" bit adds little value (default settings should be...well, default), and may sound confusing. @prcr thoughts?

Reply:
Indeed, the example use case is meant to instruct users to use the GitHub Action in the simplest (most straightforward) way possible, without going into details about extra or more advanced configurations.

So I do think that mentioning that in these instructions we'll be using the default settings is appropriate. :+1: